javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+
+function filterOutStrings(values) {
+  var numOnly = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This function is to filter out strings, not collect the numbers 🤔 . What if I gave you an array that included booleans or something too?